pokemon-minisite/src/app/components/login/login.component.html

```diff
@@ -0,0 +1,16 @@
+<body>
+  <main class="box">
+      <h2>Login</h2>
```

Comment:
Use div and give it style. without h2...

Reply:
done